### PR TITLE
fix(wsgi): handle no root span

### PR DESCRIPTION
**Pull request by Kyle-Verhoog**
# Bug fix

## Description

It is possible for `tracer.current_root_span()` to return None which was
not handled here.

## Fix

Handle the possiblity that the current root span is None.

# Checklist
- [ ] Regression test added; or
- [x] Description of manual testing performed and explanation is included in the code and/or PR.
- [ ] Added to the correct milestone.

### CODE REVIEW

**Comment by Kyle-Verhoog** [comment]
@mergifyio backport 0.50

**Review by jd — DISMISSED** [review]
Typically the kind of issue we would catch with `mypy` if we had typing here :)

File: ddtrace/contrib/wsgi/wsgi.py [hunk 1]
```diff
@@ -97,9 +97,10 @@ def __init__(self, application, tracer=None, span_modifier=default_wsgi_span_mod
     def __call__(self, environ, start_response):
         def intercept_start_response(status, response_headers, exc_info=None):
             span = self.tracer.current_root_span()
-            status_code, status_msg = status.split(" ", 1)
-            span.set_tag("http.status_msg", status_msg)
-            trace_utils.set_http_meta(span, config.wsgi, status_code=status_code, response_headers=response_headers)
+            if span:
```
Comment (jd) [inline ddtrace/contrib/wsgi/wsgi.py:100]:
```suggestion
            if span is not None:
```

**Comment by Kyle-Verhoog** [comment]
Since there are no library dependencies we actually could/should add typing for the WSGI middleware 🤔

**Review by Yun-Kim — APPROVED** [review]

**Review by jd — APPROVED** [review]

**Pull request merged**